clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5390,6 +5390,7 @@ def AtomicCmpXchg : CIR_Op<"atomic.cmp_xchg",
                        CIR_AnyType:$desired,
                        Arg<MemOrder, "success memory order">:$succ_order,
                        Arg<MemOrder, "failure memory order">:$fail_order,
+                       DefaultValuedAttr<I64Attr, "0">:$alignment,
```

Comment:
I suggest using `OptionalAttr<I64Attr>` instead as the constraint. See the definition of `cir.load`.

Also, please update the assembly format accordingly and include the alignment information.

Reply:
+1